### ISSUE TITLE
Live users get kicked out of meetings when streaming more than one meeting in docker containers.

### ISSUE THREAD

**Issue by beri-sela** [aau-zid/BigBlueButton-liveStreaming#21]
When streaming several meetings in separate docker containers, both users get kicked out shortly after the second stream starts, thereby ending both streams. 

**Steps to reproduce the behavior:**
1. Create 2 BigBlueButton meetings and 2 stream URLs on YouTube(you will need 2 accounts).
2. Create 2 folders e,g Stream1 and Stream2 and put a docker-compose.yml file in each folder. 
3. Set the contents of each docker-compose.yml file, including meeting ID, and stream URL.
4. In folder Stream1, start the first stream with `docker-compose up -d` and observe that Live user joins the meeting and the first stream start.
5. In folder Stream2, start the second stream with `docker-compose up -d` and observe that Live user joins the meeting for a few seconds but stream 2 does not start.
6. Both users get kicked out of meetings and stream 1 stops.

**Expected behavior**

 Both meetings should be streamed to respective URLs and Live users should not get kicked out.

**Desktop :**
 - Browser [Chrome]
 - Version [81.0.4044.138]

**Comment by mtsonline** [comment]
Hi and thanks for your feedback.
did you have a look at the load of your server? I guess two streams on one server will not work.

The streaming causes very high cpu usage - to give some recommendations, we are evaluating what the minimum requirements are.

we also try to improve it more in future versions.
 

in our tests four webcams and one stream needed 8 cpu cores and 4 GB of memory.

we will test if there is some additional problem with two streams on one server though.

cheers

**Comment by mbile** [comment]
Hello @beri-sela,

Can you please guide me on how to get stream URLs from my youtube channel. I am trying to implement the bbb-liveStreaming solution but can't find the RTMP server URL.

Thanks

**Comment by snehgin** [comment]
@mtsonline @beri-sela  I am getting this issue even while streaming only one meeting provided the BBB server and docker for streaming are on different machines.

**Comment by mtsonline** [comment]
what kind of server and how many cpu/memory do you have?

**Comment by snehgin** [comment]
> what kind of server and how many cpu/memory do you have?

8 CPU, 20GB. Please see the screenshot. I am facing the issue with both BBB and streaming on same machine also now.
![bbb_server](https://user-images.githubusercontent.com/17470798/94547340-c9c84180-026c-11eb-9f9e-e6267599df9d.JPG)

**Comment by snehgin** [comment]
@mtsonline Is there anything like inactivity timeout, the live stream user gets kicked out after approx. 30 seconds after joining?

**Comment by mtsonline** [comment]
I think you have a different problem - your users do not get kicked out when you have more than one streaming do you?

Could you have a look at other closed issues, you seem to have the same problem as others already had. ;-)

does it start to stream at all? sounds as if your live user does not get into the room?
what do you see when you open the join url shown in the docker logs (you can start without -d to see them on the commandline)

**Comment by snehgin** [comment]
@mtsonline Thanks for the inputs I checked bug 78. 

 - The console error is same as mentioned in #78 
 - The join meeting URL printed in console works fine from browser. I can see the 'Live' user active for some time and then disappears.
 - I have tried commenting the intro video section as well.
Do I need to enable any specific ports??

![bbb_error](https://user-images.githubusercontent.com/17470798/94660083-deffa780-0322-11eb-9694-d3434e7f0466.JPG)
